.ci/Jenkinsfile

```diff
@@ -47,10 +47,10 @@ pipeline {
       }
       post {
         always {
-          archiveArtifacts(allowEmptyArchive: true, artifacts: '.elastic-package/tmp/test/reports/*.xml')
```

Comment:
I'd use wrap the block into a dir step, so that we capture the base dir (defined in https://github.com/elastic/elastic-package/blob/master/.ci/Jenkinsfile#L8):

```groovy
always {
          dir("${BASE_DIR}"){
                    archiveArtifacts(allowEmptyArchive: true, artifacts: 'build/test-results/*.xml')
                    junit(allowEmptyResults: false,
                          keepLongStdio: true,
                          testResults: "build/test-results/*.xml")
          }
}
```

Reply:
Thanks! Done in b1feb7c.

Reply:
@mdelapenya I made this change and it seems to have broken the build (it was green before the build). Did I do something wrong: b1feb7c?